src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -31,7 +31,7 @@ col-sm-2
             {{- form_widget(form) -}}
             {{- form_errors(form) -}}
         </div>
-{##}</div>
+    </div>
```

Comment:
I think this was done to avoid getting the ``\n    `` in the output, as there is no Twig markup to hold some whitespace control here otherwise